bin/test

```diff
@@ -18,14 +18,25 @@ var testRunFolder = '.vscode-test';
 var testRunFolderAbsolute = path.join(process.cwd(), testRunFolder);
 
 var version = process.env.CODE_VERSION || '*';
+var isInsiders = version === 'insiders';
 
 var downloadPlatform = (process.platform === 'darwin') ? 'darwin' : process.platform === 'win32' ? 'win32-archive' : 'linux-x64';
 
-var windowsExecutable = path.join(testRunFolderAbsolute, 'Code');
-var darwinExecutable = path.join(testRunFolderAbsolute, 'Visual Studio Code.app', 'Contents', 'MacOS', 'Electron');
-var linuxExecutable = path.join(testRunFolderAbsolute, 'VSCode-linux-x64', 'code');
-if (['0.10.1', '0.10.2', '0.10.3', '0.10.4', '0.10.5', '0.10.6', '0.10.7', '0.10.8', '0.10.9'].indexOf(version) >= 0) {
-    linuxExecutable = path.join(testRunFolderAbsolute, 'VSCode-linux-x64', 'Code');
+var windowsExecutable;
+var darwinExecutable;
+var linuxExecutable;
+
+if(isInsiders) {
+    windowsExecutable = path.join(testRunFolderAbsolute, 'Code - Insiders');
+    darwinExecutable = path.join(testRunFolderAbsolute, 'Visual Studio Code - Insiders.app', 'Contents', 'MacOS', 'Electron');
+    linuxExecutable = path.join(testRunFolderAbsolute, 'VSCode-linux-x64', 'code-insiders');
```

Comment:
I intentionally dropped the special version handling for older versions for the Insiders stream.